utils/forest_utils/src/io/progress_bar.rs

```diff
@@ -16,6 +27,17 @@ pub enum ProgressBarVisibility {
     Never,
 }
 
+impl ProgressBarVisibility {
+    /// Checks if stdout is a TTY
+    pub fn should_display(&self) -> bool {
```

Comment:
I guess we could add a test that ensures it's false during unit tests. Not sure if we can instrument our tests to check the other condition. Not a big deal, though.